drivers/sensor/lis2dh/lis2dh_trigger.c

```diff
@@ -48,8 +48,9 @@ static int lis2dh_trigger_drdy_set(const struct device *dev,
 	/* cancel potentially pending trigger */
 	atomic_clear_bit(&lis2dh->trig_flags, TRIGGED_INT1);
 
-	status = lis2dh->hw_tf->update_reg(dev, LIS2DH_REG_CTRL3,
-					   LIS2DH_EN_DRDY1_INT1, 0);
+	status = lis2dh->hw_tf->update_reg(dev,
+		LIS2DH_REG_CTRL3,
+		LIS2DH_EN_DRDY1_INT1, 0);
```

Comment:
argument should be aligned to 'dev', like it was before. Do it globally

Reply:
it was a mixup before, some with alignment to the first parameter:
https://github.com/zephyrproject-rtos/zephyr/blob/4129b10b6900133e41829893c36a01527f293548/drivers/sensor/lis2dh/lis2dh_trigger.c#L150
and some with a tab indentation:
https://github.com/zephyrproject-rtos/zephyr/blob/4129b10b6900133e41829893c36a01527f293548/drivers/sensor/lis2dh/lis2dh_trigger.c#L155

Now I changed it globally to the tab indentation. I think it is a matter of taste

Reply:
> Now I changed it globally to the tab indentation. I think it is a matter of taste

AFAIK it's a requirement.

Reply:
my problem was that I had TAB set to 4 spaces, corrected it now